# CVE-2022-34981


## Metadata
| Published | Last Modified | Assigner | Version | Format |
| --------- | ------------- | -------- | ------- | ------ |
| 2022-07-22 15:15:00 | 2022-07-29 03:13:00 | cve@mitre.org | 4.0 | MITRE | 

## Description
| Language | Description |
| -------- | ----------- |
| en | The PyCrowdTangle package in PyPI before v0.0.1 included a code execution backdoor inserted by a third party. |

## Configurations
| Vulnerable | CPE 23 URI | CPE Name | Operator |
| ---------- | ---------- | -------- | -------- |
| True | cpe:2.3:a:pycrowdtangle_project:pycrowdtangle:*:*:*:*:*:pypi:*:* | [] | OR |

## CVSSv3 Information
| Score | Severity | Impact Score | Exploitability Score | Vector String | Version | Attack Vector | Attack Complexity | Privileges Required | User Interaction | Scope | Confidentiality Impact | Integrity Impact | Availability Impact |
| ----- | -------- | ------------ | -------------------- | ------------- | ------- | ------------- | ----------------- | ------------------- | ---------------- | ----- | ---------------------- | ---------------- | ------------------- |
| 9.8 | CRITICAL | 5.9 | 3.9 | CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H | 3.1 | NETWORK | LOW | NONE | NONE | UNCHANGED | HIGH | HIGH | HIGH |

## CVSSv2 Information
*** Not Provided/Calculated ***

## Problem Type
NVD-CWE-noinfo

## References
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
https://github.com/UPB-SS1/PyCrowdTangle/issues/1|https://github.com/UPB-SS1/PyCrowdTangle/issues/1|MISC|['Exploit', 'Issue Tracking', 'Third Party Advisory']
https://pypi.org/project/PyCrowdTangle/|https://pypi.org/project/PyCrowdTangle/|MISC|['Product']
http://pypi.doubanio.com/simple/request|http://pypi.doubanio.com/simple/request|MISC|['Not Applicable']
